chains/hashkey.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "hashkey",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://explorer.hsk.xyz/api"
+    },
+    "browserUrl": "https://explorer.hsk.xyz/"
+  },
+  "id": "177",
+  "name": "HashKey",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://hashkeychain-mainnet.alt.technology"
```

Comment:
I also see `https://mainnet.hsk.xyz` at chainlist. I couldn't find any mainnet docs, where did you get this from?

Reply:
This was from [slack](https://api3workspace.slack.com/archives/C075C5780HJ/p1734374964887009?thread_ts=1734288910.652569&cid=C075C5780HJ)

Reply:
It has changed since then https://docs.hsk.xyz/build-on-hashkey-chain/network-infos#connectting-to-mainnet